sonic-slave-buster/Dockerfile.j2

```diff
@@ -310,7 +310,10 @@ RUN apt-get update && apt-get install -y \
 # For SWI Tools
         python-m2crypto \
 # For build dtb
-	device-tree-compiler
+	device-tree-compiler \
+# For secure boot key signing and import
+	efitools \
+	mokutil
```

Comment:
>mokutil [](start = 1, length = 7)

Do you need mokutil in runtime to import key? sonic-slave is only for build.

Reply:
It would be beneficial to have mokutil during runtime

Reply:
Your comment said "importing secure boot keys", and I guess it is for `mokutil`. It is confusing, because sonic-slave-buster is a build environment (docker container). And it is different from a running SONiC. Why you need it in build time and how do you use it?

---
In reply to: [480381168](https://github.com/Azure/sonic-buildimage/pull/5286#discussion_r480381168) [](ancestors = 480381168)